frontend/src/static/css/product.scss

```diff
@@ -190,4 +191,128 @@
       }
     }
   }
+
+  .form-new-register {
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+    background: #fff;
+    box-shadow: 0px 4px 10px rgba(157, 157, 157, 0.25);
+    border-radius: 14px 0px 0px 14px;
+    border: none;
+    border-right: 1px solid #D7D7D7;
+
+    form {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      width: 100%;
+      height: 100%;
+
+      .input-photo {
+        display: none;
+      }
+      
+      .title {
+        color: #515151;
+        margin: 1% 0;
+        width: 90%;
+        font-size: 14px;
+        font-weight: 500;
+      }
+
+      input {
+        outline: none;
+        width: 90%;
+        height: 5%;
+        padding-left: 1%;
+        margin-bottom: 2%;
+        border: 1px solid #949494;
+        border-radius: 10px;
+        font-size: 16px;
+        color: #333;
+        transition: .24s;
+
+        &:focus {
+          transition: .24s;
```

Comment:
when you use the transition in the selector of component, don't need set it in event selector, so, you can remove it

Reply:
Yes, but if you set the transition in input, its works to all events